dataland-frontend/tests/component/testUtils/existenceChecks.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Checks if an image is present
+ * @param name the "alt" identifier of the image
+ * @param file the file the image is expected to display
+ */
+export function checkImage(name: string, file: string): void {
```

Comment:
filename fände ich hier besser. Außerdem überprüfst du hier den Filename nur auf einen Substring. Für einen strikten Substring gibt es hier aber gar keinen use case.
.should("have.attr", "src", file) sollte eine passende Alternative sein.

Reply:
geändert